.travis.yml

```diff
@@ -13,7 +13,7 @@ before_install:
   - sudo /etc/init.d/mysql stop
   - git clone https://github.com/FAForever/faf-stack.git faf-stack
       && pushd faf-stack
-      && git checkout 8e9e3bd
+      && git checkout v19.8.12
```

Comment:
It would be cool, if there will be something like `latest` tag.

Reply:
But it's not for this PR

Reply:
Hm. I'm not sure if that's a good idea, because then it would be possible for changes in faf-stack to break the server travis build. I prefer to do it manually to make sure that builds don't randomly break